staging/src/k8s.io/kubectl/pkg/explain/v2/templates/plaintext.tmpl

```diff
@@ -2,13 +2,33 @@
 {{- $prefix := (ternary "/api" (join "" "/apis/" $.GVR.Group) (not $.GVR.Group)) -}}
 
 {{- /* Search both cluster-scoped and namespaced-scoped paths for the GVR to find its GVK */ -}}
-{{- /* Looks for path /apis/<group>/<version>/<resource> or /apis/<group>/<version>/<version>/namespaces/{namespace}/<resource> */ -}}
+{{- /* Also search for paths with {name} component in case the list path is missing */ -}}
```

Comment:
Isn't `kubectl explain bindings` going to openapiv3?, why we are only making changes on openapi/v2 templates?. I'd expect changes on v3, why am I missing?

Reply:
It's kinda confusing, but `explain/v2` is the v2 package of explain which is responsible for openapi v3.

Reply:
Ah.. now it makes sense :)